stream-api/src/main/kotlin/fund/cyber/markets/api/common/WebSocketCommandsParser.kt

```diff
@@ -48,12 +49,12 @@ class WebSocketCommandsParser(
     }
 
     private fun parseTradesSubscription(jsonMessage: JsonNode): WebSocketCommand {
-        var pairs = jsonMessage["pairs"]?.toList()
-                ?.map { pairLabel -> TokensPair.fromLabel(pairLabel.asText(), "_") }
-                ?.toList()
-        var exchanges = jsonMessage["exchanges"]?.toList()
-                ?.map { exchange -> exchange.asText() }
-                ?.toList()
+        val pairs = jsonMessage["pairs"].toList()
+                .map { pairLabel -> TokensPair.fromLabel(pairLabel.asText(), "_") }
+                .toList()
+        val exchanges = jsonMessage["exchanges"].toList()
+                .map { exchange -> exchange.asText() }
+                .toList()
```

Comment:
no need to use `toList()` method. 
May be better to return "safecall" after `jsonMessage["..."] `? For example
`jsonMessage["exchanges"]?.map { exchange -> exchange.asText() } ?: emptyList()`